### DISCUSSION TITLE
Can you add a class mark?

### DISCUSSION THREAD

**Discussion by zoy-l** [heroui-inc/heroui#2879] [General]
Just like mui, each state has a className 
<img width="600" alt="image" src="https://github.com/nextui-org/nextui/assets/15136058/33ea4508-d4ee-4e58-8e54-62c498216c42">
Actually, these classNames are not styled, they are intended for external use, and can be easily overridden

**Comment by wingkwong** [comment]
We don't do in this way. You may take a look how to [override styles](https://nextui.org/docs/customization/override-styles). Behind the scene we use tailwind variants to build the styles and efficiently merge those conflicting ones.
